Microsoft.Toolkit.Mvvm/ComponentModel/ObservableObject.cs

```diff
@@ -0,0 +1,339 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable SA1512
```

Comment:
removing (empty) line 9 would remove this need for this.

Reply:
Right, I just liked the blank line after the second comment to make it look more like a second header rather than a comment for the code below. I agree though that not having the `pragma` to disable the warning there would feel cleaner. Not sure which would be best 🤔

Reply:
Should we just add it to the ignore in the project file?